packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -87,6 +87,14 @@ export type BaseInputProps = FormInputLabelProps &
      * For React Native this will call `onEndEditing` event since we want to get the last value of the input field
      */
     onBlur?: FormInputOnEvent;
+    /**
+     * Ignores the next blur event
+     */
+    shouldIgnoreBlurAnimation?: boolean;
```

Comment:
Is it only the "next" blur event? Perhaps the prop should be named appropriately.

Reply:
so the prop itself doesn't handle next blur. We're passing `setShouldIgnoreBlurAnimation` and we set it to false so that makes it only ignore next blur. Lemme rephrase the jsdoc comment instead.

Reply:
done